froide/foirequest/documents.py

```diff
@@ -76,6 +76,12 @@ def prepare_public(self, obj):
     def prepare_campaign(self, obj):
         return obj.campaign_id
 
+    def prepare_first_message(self, obj):
+        return obj.first_message
+
+    def prepare_last_message(self, obj):
+        return obj.last_message
+
```

Comment:
Why a seperate function for `last_message` as well?

Reply:
Ha, turns out you don't need this or `prepare_first_message`.